app/presenters/tree_builder_vandt.rb

```diff
@@ -2,7 +2,7 @@ class TreeBuilderVandt < TreeBuilder
   include TreeBuilderArchived
 
   def tree_init_options(_tree_name)
-    {:leaf => 'VmOrTemplate'}
+    {:leaf => 'ManageIQ::Providers::InfraManager::VmOrTemplate'}
```

Comment:
Thanks. This is what I had in mind, but wasn't able to get the last bit working.
Think this value gets cached in the session and sticks around.